toss/fitness/fitness_functions.py

```diff
@@ -1,98 +1,121 @@
+import sys
+sys.path.append("..")
+sys.path.append("../..")
+
 # Core packages
 import numpy as np
+from typing import Callable
+from fitness.fitness_function_enums import FitnessFunctions
+from fitness.fitness_function_utils import _compute_squared_distance
+from fitness.fitness_function_utils import estimate_covered_volume
+
+
+def get_fitness_function(chosen_function: FitnessFunctions) -> Callable:
+    """ Returns user specified fitness function.
+    Args:
+        chosen_function (FitnessFunctions): Chosen fitness function defined in enum class FitnessFunctions.
 
-def get_fitness_function(chosen_function):
-    """ 
-    Returns user specified fitness function.
+    Returns:
+        (callable): The correct fitness function correspondning to chosen_function.
     """
 
-    if chosen_function.value == 1:
-        return TargetAltitudeDistance
+    if chosen_function == FitnessFunctions.TargetAltitudeDistance:
+        return target_altitude_distance
     
-    elif chosen_function.value == 2:
-        return CloseDistancePenalty
+    elif chosen_function.value == FitnessFunctions.CloseDistancePenalty:
+        return close_distance_penalty
     
-    elif chosen_function.value == 3:
-        return FarDistancePenalty
+    elif chosen_function.value == FitnessFunctions.FarDistancePenalty:
+        return far_distance_penalty
     
-    elif chosen_function.value == 4:
-        return CoveredVolume
+    elif chosen_function.value == FitnessFunctions.CoveredVolume:
+        return covered_volume
     
-    elif chosen_function.value == 5:
-        return CoveredVolumeFarDistancePenalty
+    elif chosen_function.value == FitnessFunctions.CoveredVolumeFarDistancePenalty:
+        return covered_volume_far_distance_penalty
 
 
-def TargetAltitudeDistance(args, positions: np.ndarray, timesteps: None) -> float:
+def target_altitude_distance(args, positions: np.ndarray, timesteps: None) -> float:
     """ Computes average distance to target altitude for satellite positions along the trajectory.
     Args:
-        args (dotmap.DotMap): Dotmap with information on target squared altitude of spacecraft.
-        positions (np.ndarray): A set of positions along the trajectory.
-        timesteps (None): Time values for each position. 
+        args (dotmap.DotMap): Dotmap with information on radius of outer bounding sphere.
```

Comment:
why a dotmap? you only use one float, just pass that? also avoids the very generic `args`